sagemaker-keyspaces/SageMaker_Keyspaces_ml_example.ipynb

```diff
@@ -0,0 +1,469 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Connect to Cassandra Database from Sagemaker Noteboo using temporarly credentials fromt the Role. 
</pre><p>Typos: <code>Noteboo</code> -&gt; <code>Notebook</code> , <code>temporarly</code> -&gt; <code>temporary</code> ﻿, <code>fromt</code>  -&gt; <code>from</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Good catch.  Thanks </p>